File: Python Course/Software Development Module - in Python/Revision/Exam2017/SalaryIncrease/EmployeeInfo.py
"""
EmployeeInfo.py
Billy Cussen
01/11/2020
"""

class myEmployeeInfo:
    def __init__(self, position, years, salary):
        self.position = position
        self.years = years
        self.salary = salary

    def __init__(self):
        self.position = ""
        self.years = 0
        self.salary = 0.0

    def setPosition(self, position):
        self.position = position

    def setYears(self, years):
        self.years = years
    
    def setSalary(self, salary):
        self.salary = salary

    def getPosition(self):
        return self.position

    def getYears(self):
        return self.years

    def getSalary(self):
        return self.salary

File: Java/Software Development Module - Java/Revision/Exam2017/EmployeeInfo.java
/*
EmployeeInfo.java
Billy Cussen
29/08/2020
*/

public class EmployeeInfo{

    private String position;
    private int years;
    private double salary;

    //Constructor
    public EmployeeInfo(String position, int years, double salary){
        this.position = position;
        this.years = years;
        this.salary = salary;
    }

    public EmployeeInfo(){
        this.position = new String();
        this.years = 0;
        this.salary = 0.0;
    }

    //Setters
    public void setPosition(String position){
        this.position = position;
    }

    public void setYears(int years){
        this.years = years;
    }

    public void setSalary(double salary){
        this.salary = salary;
    }

    //Getters
    public String getPosition(){
        return this.position;
    }

    public int getYears(){
        return this.years;
    }

    public double getSalary(){
        return this.salary;
    }
}